custom-scope-handler/src/main/java/org/wso2/custom/scope/handler/internal/CustomScopeHandlerDataHolder.java

```diff
@@ -0,0 +1,45 @@
+/*
+ *   Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *   WSO2 Inc. licenses this file to you under the Apache License,
+ *   Version 2.0 (the "License"); you may not use this file except
+ *   in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+package org.wso2.custom.scope.handler.internal;
+
+import org.wso2.carbon.user.core.service.RealmService;
+
+public class CustomScopeHandlerDataHolder {
+    private static CustomScopeHandlerDataHolder instance = new CustomScopeHandlerDataHolder();
```

Comment:
Add a new line

Reply:
fixed at https://github.com/wso2-incubator/samples-is/pull/17